slowfast/config/defaults.py

```diff
@@ -731,9 +731,6 @@
 # Number of GPUs to use (applies to both training and testing).
 _C.NUM_GPUS = 1
 
-# Maximum number of GPUs available per node (unlikely to need to be changed)
```

Comment:
There is one reference of MAX_GPUS_PER_NODE left here that needs to be removed: https://github.com/lyttonhao/SlowFast-FSDP/blob/main/slowfast/models/build.py#L35